classes/class-wc-connect-shipping-label.php

```diff
@@ -396,7 +403,9 @@ private function calculate_should_show_meta_box() {
 			}
 
 			$dest_address = $order->get_address( 'shipping' );
-			if ( ( $dest_address['country'] && 'US' !== $dest_address['country'] )
+			if ( ( $dest_address['country'] && 'US' !== $dest_address['country']
+				//special case - include PR since it's treated as domestic
+				&& 'PR' !== $dest_address['country'] )
```

Comment:
What do you think about using a class property for this? (to represent supported countries)